Week1/SantiagoCollazosBarrera/Functional/functional/src/test/java/com/globant/bootcamp/functional/functional/katas/bootcampSession1/Streams.java

```diff
@@ -0,0 +1,17 @@
+package com.globant.bootcamp.functional.functional.katas.bootcampSession1;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+
+public class Streams {
+
+
+    @Test
+    public void stream1() {
+
+        System.out.println(Arrays.asList("juan","pedro","juan","trump").stream().distinct().map(a -> a+"yahoo").count());
```

Comment:
Extract this method to delete duplicated code

Reply:
So sorry, I worked in other project in my local machine, and I didn't change the code in the git project, at night I'm going to update it and commit again. Again, my apologizes

Reply:
I was checking that information and that code came it with the project, I think that was an example in the presentation